source/conf/cmake/crownstone.src.cmake

```
@@ -91,8 +91,8 @@ IF (BUILD_TWI)
 	LIST(APPEND FOLDER_SOURCE "${SOURCE_DIR}/drivers/cs_Twi.cpp")
 ENDIF()
 
-IF (BUILD_RSSI_DATA_TRACKER)
-	LIST(APPEND FOLDER_SOURCE "${SOURCE_DIR}/localisation/cs_RssiDataTracker.cpp")
+IF (RSSI_DATA_TRACKER_ENABLED)
```

These configs were changed by anne, to all start with BUILD_

```suggestion
IF (BUILD_RSSI_DATA_TRACKER)
```